src/Compilers/Test/Core/Platform/Desktop/DesktopRuntimeEnvironment.cs

```diff
@@ -334,6 +335,15 @@ public void Verify(Verification verification)
                 {
                     throw new Exception("Verification failed", ex);
                 }
+
+                if (verification.PEVerifyMessage != null)
+                {
+                    // remove token values to avoid dependency on on metadata ordering
```

Comment:
It's deterministic, but the tokens are not interesting since the message already contains the qualified member name.
The tokens change if you change the code in any way, so you can't have multiple tests reusing the same baseline PE verify messages.